setup.cfg

```diff
@@ -27,6 +27,7 @@ addopts =
 filterwarnings =
     # error out on our deprecation warnings - ensures the code and tests are kept up-to-date
     error::pytorch_lightning.utilities.warnings.LightningDeprecationWarning
+    error::FutureWarning
```

Comment:
will it start popping up test failures in tests in case any of the dependencies start raising this error internally?

Reply:
Yes. But this is good to allow us to be on top of user-facing changes to those dependencies.

Reply:
okay.. but still we can't say the same for dependencies of a dependency.
CI looks green now, but there is a possibility we might end up removing that in the future if that happens.

I hope we are not removing `LightningDeprecationWarning`?

Reply:
We can ignore here warnings for specific modules or libraries if that happens.